tests/test_fairness_binary.py

```diff
@@ -850,7 +850,7 @@ def test_theil_boundaries(self):
 
     @staticmethod
     def extract_metric_from_df(metric_name, df, attribute='Value'):
-        return df.loc[df.index == metric_name, attribute].values.squeeze()
+        return float(df.loc[df.index == metric_name, attribute].values.squeeze())
```

Comment:
Wow, thanks for identifying and fixing this! I wonder what was being returned from this function originally

Reply:
thank you! looks like a ndarray that was returned

Reply:
nice catch!